src/IssueLabeler/IssueLabeler.csproj

```
@@ -21,7 +21,7 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
     <None Update="local.settings.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

```suggestion
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```
This was unintentional.   Will either commit the suggestion or include with changes pushed.